Builds/levelization/results/loops.txt

```diff
@@ -4,6 +4,9 @@ Loop: test.jtx test.toplevel
 Loop: test.jtx test.unit_test
   test.unit_test == test.jtx
 
+Loop: xrpl.basics xrpl.beast
+  xrpl.basics > xrpl.beast
+
```

Comment:
This is very disappointing to me. Wondering what you think of some alternatives.

a. We cannot just directly use `ALWAYS_OR_UNREACHABLE` in `xrpl/beast/` because it is not defined on non-Linux platforms.
b. We could write our own `<cassert>` and put it on the include path before the standard library.
c. We could not rewrite assertions within `xrpl/beast/`.
d. We could move the instrumentation header into `xrpl/beast/`.
e. We could add a new module, at a level beneath the `beast` module, that includes the instrumentation header.

Reply:
I like d. Would that be ok for you as well ?